AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -48,6 +48,8 @@ gds.it###oggiinedicola
 namu.wiki##div[class="c"][data-v-37880e75]
 board.namu.wiki,arca.live###newsRank
 arca.live###namuRank
+! https://github.com/AdguardTeam/AdguardFilters/issues/79295
+toroox.com#$#.box-main > .is-table-row > .col-md-6 { width: 0% !important; }
```

Comment:
I think it's better to user these two rules:
```
toroox.com#?#.box-main > div.row > div.col-md-6:has(> div.link-details > div.panel)
toroox.com##.link-details > div.panel
```

Reply:
What is the reason to use css-rule here? `toroox.com##.link-details > div.panel` will work on all platforms

Reply:
There is a same rule already, better add to that rule instead of creating another one.

Reply:
I guess that's it.

Reply:
@Sergey-Lyapin